evmbin/src/display/json.rs

```diff
@@ -93,26 +141,30 @@ impl vm::Informant for Informant {
 					println!("{}", trace);
 				}
 
-				let success_msg = json!({
-					"output": format!("0x{}", success.output.to_hex()),
-					"gasUsed": format!("{:#x}", success.gas_used),
-					"time": display::as_micros(&success.time),
-				});
+				let message_success =
+					MessageSuccess {
+						output: &format!("0x{}", success.output.to_hex()),
+						gas_used: &format!("{:#x}", success.gas_used),
+						time: &display::as_micros(&success.time),
+					}
+				;
 
-				println!("{}", success_msg)
+				println!("{:?}", message_success);
```

Comment:
Maybe `info!`?

Reply:
Same as earlier, we don't want any additional formatting (time, module, etc) and need raw JSON on stdout.